.tsconfig

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
**question** Is this configuration the default one suggested by typescript? If not, do you know if is there any?

Reply:
The only reference that I found is this -> https://www.typescriptlang.org/docs/handbook/tsconfig-json.html , it gives us different type of basic configurations based on the node version of the project.

Reply:
I guess it points to this repository: https://github.com/tsconfig/bases/. It's the one you mean right?

Reply:
I think it's fine the configuration, just wondered where we got these from!

Reply:
do you mean my configuration or the ones from the documentation?

Reply:
the one in this MR is fine :)